uplink/storage/streams/store.go

```diff
@@ -437,19 +437,24 @@ func (s *streamStore) Delete(ctx context.Context, path Path, pathCipher storj.Ci
 		return err
 	}
 
+	var errlist errs.Group
 	for i := 0; i < int(numberOfSegments(&stream, &streamMeta)-1); i++ {
 		currentPath, err := createSegmentPath(ctx, int64(i), path.Bucket(), encPath)
 		if err != nil {
-			return err
+			errlist.Add(err)
```

Comment:
I took a look at `createSegmentPath` and I'm wondering if these errors should be skipped.

Reply:
this logic will be changed by #2640 so I don't think we need to care too much about it